flow-typed/mobx.js

```diff
@@ -127,12 +127,12 @@ export interface IObservableArray<T> extends Array<T> {
     clear(): T[];
     replace(newItems: T[]): T[];
     find(
-        predicate: (item: T, index: number, array: Array<T>) => boolean,
+        predicate: (item: T, index: number, array: Array<T>) => mixed,
```

Comment:
I don't know flow that much, but shouldn't it return T instead of mixed?

Reply:
Not according to https://github.com/facebook/flow/commit/00be88084a2ef41fadce99352b15f683f3c67f36